.github/workflows/run-CI.yml

```diff
@@ -6,20 +6,24 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python: [3.7, 3.8, 3.9, '3.10']
+        python: [3.7, 3.8, 3.9, '3.10', 3.11]
         exclude:
           - os: macos-latest
             python: 3.7
           - os: macos-latest
             python: 3.8
           - os: macos-latest
             python: 3.9
+          - os: macos-latest
+            python: 3.10
```

Comment:
Probably here the quotation marks will also be necessary

Reply:
Commit [e688555](https://github.com/WillianFuks/tfcausalimpact/pull/80/commits/e688555b6a4949624e4d8d0740b81dfec1debbd4) applies your suggestion.